Base/Python/slicer/tests/test_slicer_util_getNodes.py

```diff
@@ -1,11 +1,24 @@
 import unittest
 import slicer
 import slicer.util
+import vtk
 
 class SlicerUtilTest(unittest.TestCase):
     def setUp(self):
         slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume1")
         slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume2")
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume")
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume")
+
+    def test_getNodesMultipleNodesSharingName(self):
+
+        self.assertTrue("Volume" in slicer.util.getNodes("Volume"))
+        self.assertTrue("Volume" in slicer.util.getNodes("Volume",useLists=True))
+
+        self.assertEqual(slicer.util.getNodes("Volume").keys(), ["Volume"])
+        self.assertEqual(isinstance(slicer.util.getNodes("Volume")["Volume"], vtk.vtkObject), True)
```

Comment:
This line, and similar lines, can be written as [`self.assertIsInstance(...)`](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIsInstance).